data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/JsonImport.java

```diff
@@ -32,16 +28,7 @@
  */
 @AutoService(CracImporter.class)
 public class JsonImport implements CracImporter {
-
-    private static final String CRAC_FILE_SCHEMA_JSON = "/CracSchema.json";
```

Comment:
You can maybe delete this file if you delete its usages.

Reply:
ok